### PR TITLE
feat(toolbar & sidebar): 11734 - create tooltips for toolbar and change them for sidebar

### PR DESCRIPTION
**Pull request by IlyaIzr**
Toolbar tooltip on hover (desktop only)
image
Collapsed sidebar tooltip on hover (desktop obly, as usual for all hover tooltips)
image

### CODE REVIEW

File: src/features/legend_panel/components/LegendPanel/LegendPanel.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ interface LegendPanelProps {
 export function LegendPanel({ layers, iconsContainerRef }: LegendPanelProps) {
   // isOpen - is an overall state of panel. Was closed stores wether the panel was closed by user -> preferred to be closed
   const [isOpen, setIsOpen] = useState<boolean>(false);
-  const [wasClosed, setWasClosed] = useState<boolean>(false);
+  const [wasClosed, setWasClosed] = useState<boolean>(true);
```
Comment (IlyaIzr) [inline src/features/legend_panel/components/LegendPanel/LegendPanel.tsx:23]: That's a minor fix as we wanted to have legend panel closed by default

File: src/features/events_list/components/EventsListPanel/EventsListPanel.tsx [hunk 2]
```diff
@@ -58,7 +58,7 @@ export function EventsListPanel({
       id: EVENT_LIST_CONTROL_ID,
       name: EVENT_LIST_CONTROL_NAME,
       title: i18n.t('event_list.title'),
-      active: false,
+      active: true,
```
Comment (IlyaIzr) [inline src/features/events_list/components/EventsListPanel/EventsListPanel.tsx:61]: also minor fix in order to restore old event_list behaviour, not related to main PR topic, sorry

**Comment by IlyaIzr** [comment]
Updated sidebar tooltip positioning accordint to task 9910
image

File: src/features/side_bar/components/SideBar/SideBar.tsx [hunk 3]
```diff
@@ -36,7 +53,14 @@ export function SideBar() {
             to={APP_ROUTES[control.id]}
             tabIndex={-1}
           >
-            <div className={s.buttonWrap} onClick={() => control.onClick()}>
+            <div
+              className={s.buttonWrap}
+              onClick={() => control.onClick()}
```
Comment (propakov) [inline src/features/side_bar/components/SideBar/SideBar.tsx]:
```suggestion
              onClick={control.onClick}
```

File: src/features/toolbar/components/Toolbar.tsx [hunk 4]
```diff
@@ -1,54 +1,36 @@
...
-              size="small"
-              className={s.toolButton}
-              variant="invert"
-            />
+      {sortByPredefinedOrder(Object.values(controls), controlsOrder).map(
+        (control, index) => (
+          <div key={control.id} className={s.toolbarItem}>
+            <div
+              className={clsx([s.buttonWrap, control.active && s.active])}
+              onClick={() => control.onClick && control.onClick(!control.active)}
```
Comment (propakov) [inline src/features/toolbar/components/Toolbar.tsx]:
```suggestion
              onClick={() => control?.onClick(!control.active)}
```

**Review by propakov — APPROVED** [review]

**Pull request merged**